src/index.html

```diff
@@ -166,63 +166,150 @@ <h1>Scaling <i>beyond</i> geospatial tech</h1>
+                  additional (or fewer) channels.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/aperture.svg" class="h-7 w-7 pb-2" />
+                <p class="font-semibold text-med">Offers data augmentation</p>
+                <p>
+                  Raster Vision supports customizable Albumentations transforms
+                  to use during training.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/diagram-sankey.svg" class="h-7 w-7 pb-2" />
+                <p class="font-semibold text-med">Custom model specification</p>
+                <p>
+                  Utilize a wide variety of compatible models from local
+                  directories using TorchHub.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/flame.svg" class="h-7 w-7 pb-2" />
+                <p class="font-semibold text-med">Based on PyTorch</p>
+                <p>
+                  Raster Vision implements deep learning functionality with
+                  PyTorch (a popular deep learning library) and Torchvision.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/cloud-word.svg" class="h-7 w-7 pb-2" />
+                <p class="font-semibold text-med">
+                  Supports running pipelines in the cloud
+                </p>
+                <p>
+                  Raster Vision provides support for running pipelines using AWS
+                  Batch.
+                </p>
+              </div>
+              <div class="tech-spec text-white">
+                <img src="images/table-cells-large.svg" class="h-7 w-7 pb-2" />
+                <p class="font-semibold text-med">
+                  Supports configurable chip reading
+                </p>
+                <p>
+                  Geospatial images tend to be too large to feed directly into a
+                  neural network and must first be broken up into smaller
+                  “chips”.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="text-center mb-8">
             <h2 class="text-white">
               Built on popular technologies & open source libraries including
             </h2>
-            <p class="lg-body text-white">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit...
+            <p class="large-body text-white pb-4">
+              AWS (S3 and Batch), PyTorch, TorchVision, Albumentations,
+              Rasterio, Shapely, Gdal and Numpy.
             </p>
             <div
-              class="grid grid-cols-2 gap-2 sm:grid-cols-4 sm:gap-x-16 max-w-xl m-auto"
+              class="grid grid-cols-2 gap-2 sm:grid-cols-4 sm:gap-x-16 max-w-xl m-auto place-items-center"
```

Comment:
```suggestion
              class="grid grid-cols-2 max-w-xs sm:grid-cols-4 sm:gap-x-16 sm:max-w-xl m-auto place-items-center"
```
These logos looked a little far apart on the screen size between mobile and small, so this just keeps them from wandering about.